actions/trickle-down-changelog/index.js

```diff
@@ -0,0 +1,119 @@
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+import * as semver from 'semver';
+import * as exec from "@actions/exec"
+import * as fs from "fs/promises";
+
+const octokit = github.getOctokit(core.getInput("token"));
+const simpleSemverRegex = /\d+\.\d+\.\d+(-.*)?/;
```

Comment:
```suggestion
const isValidVersionFormat = /\d+\.\d+\.\d+(-.*)?/;
```

Reply:
sorry, like my name better, is it important to you?

Reply:
Nope :) 
Your name is cool as well.
What `Sem` stands for?

Reply:
https://semver.org/
semantic versioning the official protocol for 1.0.0 like versions